### ISSUE TITLE
Version v0.1.5 is not working with Storybook v6.5.10

### ISSUE THREAD

**Issue by gustavoteodoro** [prismicio/prismic-next#41]
### Versions

- `@prismicio/next`: v0.1.5
- `next`: v12.3.0
- node: v18.7.0
- storybook: v6.5.10

### Reproduction

- Install the latest version of "@prismicio/next": `0.1.5`
- Install the latest version of "storybook": `6.5.10`
- Create a story of a component that uses any Prismic Image
- Try to run storybook

<details open>
<summary>Additional Details</summary>
<br>

### Package
```json
{
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "test": "jest",
    "storybook": "start-storybook -p 6006",
    "build-storybook": "build-storybook",
    "chromatic": "npx chromatic",
    "slicemachine": "start-slicemachine",
    "e2e": "cypress open",
    "e2e:headless": "cypress run",
    "codegen": "prismic-ts-codegen"
  },
  "dependencies": {
    "@prismicio/client": "6.7.1",
    "@prismicio/helpers": "2.3.3",
    "@prismicio/next": "0.1.5",
    "@prismicio/react": "2.5.0",
    "@prismicio/slice-simulator-react": "0.2.2",
    "next": "12.3.0",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "recoil": "0.7.5",
    "styled-components": "5.3.5"
  },
  "devDependencies": {
    "@babel/core": "7.19.0",
    "@prismicio/types": "0.2.3",
    "@storybook/addon-a11y": "6.5.10",
    "@storybook/addon-actions": "6.5.10",
    "@storybook/addon-essentials": "6.5.10",
    "@storybook/addon-interactions": "6.5.10",
    "@storybook/addon-links": "6.5.10",
    "@storybook/addons": "6.5.10",
    "@storybook/react": "6.5.10",
    "@storybook/testing-library": "0.0.13",
    "@storybook/theming": "6.5.10",
    "@testing-library/jest-dom": "5.16.5",
    "@testing-library/react": "13.4.0",
    "@types/jest": "29.0.1",
    "@types/node": "18.7.16",
    "@types/react": "18.0.19",
    "@types/react-dom": "18.0.6",
    "@types/styled-components": "5.1.26",
    "@typescript-eslint/eslint-plugin": "5.36.2",
    "babel-loader": "8.2.5",
    "chromatic": "6.9.0",
    "cypress": "10.7.0",
    "eslint": "8.23.1",
    "eslint-config-next": "12.3.0",
    "eslint-plugin-jsx-a11y": "6.6.1",
    "eslint-plugin-react": "7.31.8",
    "eslint-plugin-storybook": "0.6.4",
    "eslint-plugin-styled-components-a11y": "1.0.0",
    "jest": "29.0.3",
    "jest-environment-jsdom": "29.0.3",
    "jest-styled-components": "7.1.1",
    "prismic-ts-codegen": "0.1.5",
    "slice-machine-ui": "0.4.2",
    "storybook-addon-designs": "6.3.1",
    "storybook-addon-next-router": "4.0.0",
    "storybook-addon-recoil-flow": "1.3.1",
    "tsconfig-paths-webpack-plugin": "4.0.0",
    "typescript": "4.8.3"
  }
}

```

### Log
```
ERROR in ./node_modules/@prismicio/next/dist/PrismicPreview.mjs 77:30-38
Can't import the named export 'Fragment' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/PrismicPreview.mjs 80:26-40
Can't import the named export 'PrismicToolbar' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/setPreviewData.mjs 4:45-59
Can't import the named export 'cookie' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/lib/getPrismicPreviewCookie.mjs 12:17-31
Can't import the named export 'cookie' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/PrismicPreview.mjs
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/PrismicPreview.mjs 80:22-25
Can't import the named export 'jsx' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/PrismicPreview.mjs 77:25-29
Can't import the named export 'jsxs' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/PrismicPreview.mjs 17:2-17
Can't import the named export 'useEffect' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/PrismicPreview.mjs 14:17-26
Can't import the named export 'useRouter' from non EcmaScript module (only default export is available)
 @ ./node_modules/@prismicio/next/dist/index.mjs
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/index.mjs 6:0-58
Can't reexport the named export 'PrismicNextImage' from non EcmaScript module (only default export is available)
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

ERROR in ./node_modules/@prismicio/next/dist/PrismicNextImage.mjs 50:16
Module parse failed: Unexpected token (50:16)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       width: layout === "fill" ? void 0 : field.dimensions.width,
|       height: layout === "fill" ? void 0 : field.dimensions.height,
>       alt: alt ?? (field.alt || fallbackAlt),
|       loader: imgixLoader,
|       layout,
 @ ./node_modules/@prismicio/next/dist/index.mjs 6:0-58 6:0-58
 @ ./utils/prismic/index.ts
 @ ./components/quarks/RichText/RichText.tsx
 @ ./components/quarks/RichText/index.ts
 @ ./components/molecules/ShowcaseCard/ShowcaseCard.tsx
 @ ./components/molecules/ShowcaseCard/index.ts
 @ ./components/organisms/Showcase/Showcase.tsx
 @ ./components/organisms/Showcase/index.ts
 @ ./components/organisms/Showcase/Showcase.stories.tsx
 @ ./components sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined ./storybook-init-framework-entry.js ./node_modules/@storybook/react/dist/esm/client/docs/config-generated-config-entry.js ./node_modules/@storybook/react/dist/esm/client/preview/config-generated-config-entry.js ./node_modules/@storybook/addon-links/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-docs/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-actions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-backgrounds/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-measure/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-outline/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-interactions/preview.js-generated-config-entry.js ./node_modules/@storybook/addon-a11y/preview.js-generated-config-entry.js ./node_modules/storybook-addon-next-router/dist/preset/addDecorator.js-generated-config-entry.js ./node_modules/storybook-addon-recoil-flow/dist/decorators.js-generated-config-entry.js ./.storybook/preview.js-generated-config-entry.js ./generated-stories-entry.js

```

</details>

### What is expected?

The Storybook runs, loading all the Prismic Images

### What is actually happening?

The storybook run fails with the error described above

**Comment by gustavoteodoro** [comment]
It's fixed with version 0.1.6 🏆

**Comment by angeloashmore** [comment]
Thanks for closing the issue, @gustavoteodoro! 🙂 I totally forgot to reply that `v0.1.6` fixes the issue.

For anyone else coming across this issue, you can update to the latest version with the fix using the following command:

```sh
npm install @prismicio/next@latest
```
